From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: sin-ack <sin-ack@users.noreply.github.com>
Date: Mon, 17 Jul 2023 03:26:04 +0300
Subject: [PATCH] build: Set correct Zig version

The build script contains the version of zig-bootstrap's infrequently
updated vendored Zig. Since we overwrite it with Zig master we have to
patch the version too.
---
 build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/build b/build
index c6ad2b5e1b115c49432b6512215906013178f7e2..cab9e3431bfe7e9b8395888ceef52d00216b6f2c 100755
--- a/build
+++ b/build
@@ -7,7 +7,7 @@ TARGET="$1" # Example: riscv64-linux-gnu
 MCPU="$2" # Examples: `baseline`, `native`, `generic+v7a`, or `arm1176jzf_s`
 
 ROOTDIR="$(pwd)"
-ZIG_VERSION="0.15.0-dev.929+31e46be74"
+ZIG_VERSION="0.15.0-dev.1517+d62515835"
 
 TARGET_OS_AND_ABI=${TARGET#*-} # Example: linux-gnu
 
